capa/features/extractors/base_extractor.py

```diff
@@ -11,13 +11,9 @@
 import dataclasses
 from copy import copy
 from types import MethodType
-from typing import Any, Set, Dict, Tuple, Union, Iterator
+from typing import Any, Union, Iterator, TypeAlias
 from dataclasses import dataclass
 
-# TODO(williballenthin): use typing.TypeAlias directly when Python 3.9 is deprecated
-# https://github.com/mandiant/capa/issues/1699
-from typing_extensions import TypeAlias
```

Comment:
can we remove this from dependencies?

Reply:
we still use `from typing_extensions import assert_never` (once) and the proto pyi files also import `typing_extensions`

Reply:
and that :)